lib/themes/dosomething/paraneue_dosomething/scss/content/_onboarding.scss

```diff
@@ -8,6 +8,45 @@
     position: relative;
   }
 
+  &.-fullscreen {
+    position: fixed;
+    z-index: 1000;
+    top: 0;
+    left: 0;
+    width: 100vw;
```

Comment:
Mostly curious if there was a specific reason. Did you test in IE and other browsers to make sure it works right? I think support for `vw` units is better nowadays, but best to confirm it works right across multiple browsers!